nvtabular/tools/data_gen.py

```diff
@@ -0,0 +1,442 @@
+import os
+import random
+import string
+
+import cudf
+import cupy
+import numpy as np
+from cudf.core.column import as_column, build_column
+from scipy import stats
+from scipy.stats import powerlaw, uniform
+
+from ..io.dataset import Dataset
+from ..utils import device_mem_size
+
+
+class UniformDistro:
+    def create_col(self, num_rows, dtype=np.float32, min_val=0, max_val=1):
+        ser = cudf.Series(cupy.random.uniform(min_val, max_val, size=num_rows))
```

Comment:
The dtype parameter for this method seems to be ignored - should this be 
```suggestion
        ser = cudf.Series(cupy.random.uniform(min_val, max_val, size=num_rows), dtype=dtype)
```

?